py3status/modules/window_title.py

```diff
@@ -1,61 +1,67 @@
 # -*- coding: utf-8 -*-
 """
-Display window title.
+Display window title frequently.
```

Comment:
let's remove frequently it adds nothing we could say this for every module

Reply:
This was for... 
```
$ py3status modules list
  window_title           Display window title.
  window_title_async     Display window title asynchronously.
```
I revert the `[{title}]` thing in next comment.